access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accessreviewsv2-root.md

---
title: "Azure AD access reviews"
description: "Use Azure AD access reviews to configure one-time or recurring access reviews for attestation of user's access rights to Azure AD resources."
localization_priority: Normal
author: "isabelleatmsft"
ms.prod: "governance"
doc_type: conceptualPageType
---

# Azure AD access reviews

Namespace: microsoft.graph

Use [Azure AD access reviews](/azure/active-directory/active-directory-azure-ad-controls-access-reviews-overview) to configure one-time or recurring access reviews for attestation of user's access rights to Azure AD resources.

Typical customer scenarios for access reviews of group memberships and application and Azure AD role access are:

> - Customers can review and certify guest user access by using access reviews of their access to applications, Azure AD roles and memberships of groups. Reviewers can use the insights that are provided to efficiently decide whether guests should have continued access.

Reviewer: ```suggestion
- Customers can review and certify guest user access to applications, Azure AD roles, and memberships of groups. Reviewers can use the insights that are provided to efficiently decide whether guests should have continued access.
```

> - Customers can review and certify employee access to applications, Azure AD roles and group memberships with access reviews.

Reviewer: ```suggestion
- Customers can review and certify employee access to applications, Azure AD roles, and group memberships with access reviews.
```

> 
> Note that the access reviews feature, including the API, is included in Azure AD Premium P2.  The tenant where an access review is being created must have a valid purchased or trial Azure AD Premium P2 or EMS E5 subscription.

Reviewer: Text below is enough.
```suggestion
```

---
title: "Azure AD access reviews"

> description: "You can use Azure AD access reviews to configure one-time or recurring access reviews for attestation of user's access rights."

Reviewer: ```suggestion
description: "Use Azure AD access reviews to configure one-time or recurring access reviews for attestation of user's access rights to Azure AD resources."
```

localization_priority: Normal
author: "isabelleatmsft"
ms.prod: "governance"
doc_type: conceptualPageType
---

# Azure AD access reviews

Namespace: microsoft.graph

> You can use [Azure AD access reviews](/azure/active-directory/active-directory-azure-ad-controls-access-reviews-overview) to configure one-time or recurring access reviews for attestation of user's access rights.

Reviewer: ```suggestion
Use [Azure AD access reviews](/azure/active-directory/active-directory-azure-ad-controls-access-reviews-overview) to configure one-time or recurring access reviews for attestation of user's access rights to Azure AD resources.
```

Typical customer scenarios for access reviews of group memberships and application access are:

- Customers can review and certify guest user access by using access reviews of their access to applications and memberships of groups. Reviewers can use the insights that are provided to efficiently decide whether guests should have continued access.

- Customers can review and certify employee access to applications and group memberships with access reviews.

- Customers can collect access review controls into programs that are relevant for your organization to track reviews for compliance or risk-sensitive applications.

There is also a related capability for customers to review and certify the role assignments of administrative users who are assigned to Azure AD roles such as Global Administrator or Azure subscription roles.  This capability is included in [Azure AD Privileged Identity Management](privilegedidentitymanagement-root.md).

> Note that the access reviews feature, including the API, is included in Azure AD Premium P2.  The tenant where an access review is being created must have a valid purchased or trial Azure AD Premium P2 or EMS E5 subscription.

Reviewer: ```suggestion
The access reviews feature, including the API, is available only with a valid purchase or trial license of Azure AD Premium P2 or EMS E5 subscription.
```
